src/EA.Weee.Web/Areas/Scheme/Controllers/HomeController.cs

```diff
@@ -138,7 +142,19 @@ internal async Task<List<string>> GetActivities(Guid pcsId, OrganisationData org
                 if (organisationDetails.HasDirectRegistrant)
                 {
                     activities.Add(ProducerSubmissionConstant.HistoricProducerRegistrationSubmission);
-                    activities.Add(ProducerSubmissionConstant.NewContinueProducerRegistrationSubmission);
+
+                    foreach (var registrant in organisationDetails.DirectRegistrants)
```

Comment:
@will-hope98  think for the time being we just  get the first item here, which will get updated in another story. As it is we have potential to add duplicate options.

Reply:
Done